packages/embark-deploy-tracker/src/test/deploymentChecksSpec.js

```diff
@@ -0,0 +1,196 @@
+/*globals describe, it*/
+import DeploymentChecks from "../deploymentChecks";
+import TrackingFunctions from "../trackingFunctions";
+const fs = require('fs-extra');
+const sinon = require('sinon');
+const expect = require('expect.js');
+import {beforeEach, afterEach} from "mocha";
+import Logger from "embark-logger";
+require("colors");
+
+describe('embark.deploymentChecks', function () {
+  let logger = new Logger({logLevel: 'error'});
+  let events = {once: () => {}, setCommandHandler: () => {}, emit: () => {}, on: () => {}, request: () => {}, request2: () => {}};
```

Comment:
We should probably create an Events Mock in a test util file

Reply:
Agreed, along with a logger spy, so we can add tests for console output as well.